Utils/Dataflow/095_datasetInfoAMI/README

```diff
@@ -17,3 +17,20 @@ Since requesting information from AMI is a very time-consuming process,
 the supplied datasets are filtered by data format - the ones with formats
 mentioned above are included into request to AMI and their metadata are
 updated, while all the others are left unchanged.
+
+2. Requirements
+---------------
+Proper memberships and grid certificate are required to run the stage.
+This includes:
+
+1) Registering in CERN systems (discuss this with your supervisor).
+2) Obtaining a grid certificate (https://ca.cern.ch/ca/).
+3) Registering in ATLAS Virtual Organization (VO ATLAS, see first link below).
```

Comment:
'first link below' call out to a replacement with some reference like 'see [1]'; but right now it would look strange (adding ID only to the one of the links).

I would suggest one of the following variants:

```
This includes:

1) Registering in CERN systems (discuss this with your supervisor).
2) Obtaining a grid certificate (see [1]).
3) Registering in ATLAS Virtual Organization (see [2]).
4) Registering on AMI site with the grid certificate (see [3]).


Useful links:

[1] https://ca.cern.ch/ca/
[2] https://twiki.ific.uv.es/twiki/bin/view/Atlas/AtlasDataProcessingAtIFIC
[3] https://ami.in2p3.fr/pages/getting-started
```
or
```
This includes:

1) Registering in CERN systems (discuss this with your supervisor).
2) Obtaining a grid certificate (https://ca.cern.ch/ca/).
3) Registering in ATLAS Virtual Organization
   (https://twiki.ific.uv.es/twiki/bin/view/Atlas/AtlasDataProcessingAtIFIC).
4) Registering on AMI site with the grid certificate (https://ami.in2p3.fr/).
	
See also: https://ami.in2p3.fr/pages/getting-started
```

In the first variant I joined links to AMI home and AMI getting started pages; it seems like for "Register on AMI site" the getting-started link will also work good enough.

Reply:
Done.